src/app/products/recurring-deposit-products/view-recurring-deposit-product/view-recurring-deposit-product.component.html

```diff
@@ -0,0 +1,415 @@
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Minimum Deposit Term:</span>
+          <span fxFlex="60%"
+            *ngIf="recurringDepositProduct.minDepositTerm !== undefined">{{ recurringDepositProduct.minDepositTerm }}
+            &nbsp;
+            {{ recurringDepositProduct.minDepositTermType.value }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Maximum Deposit Term:</span>
+          <span fxFlex="60%"
+            *ngIf="recurringDepositProduct.maxDepositTerm !== undefined">{{ recurringDepositProduct.maxDepositTerm }}
+            &nbsp;
+            {{ recurringDepositProduct.maxDepositTermType.value }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Deposit Term In multiple of:</span>
+          <span fxFlex="60%"
+            *ngIf="recurringDepositProduct.inMultipleOfDepositTerm !== undefined">{{ recurringDepositProduct.inMultipleOfDepositTerm }}
+            -
+            {{ recurringDepositProduct.inMultipleOfDepositTermType.value }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Recurring Deposit Type:</span>
+          <span fxFlex="60%" *ngIf="recurringDepositProduct.isMandatoryDeposit"> Mandatory </span>
+          <span fxFlex="60%" *ngIf="!recurringDepositProduct.isMandatoryDeposit"> Voluntary </span>
+        </div>
+
+        <div fxFlexFill *ngIf="recurringDepositProduct.preClosurePenalInterest">
+          <span fxFlex="40%">Apply penal interest (less):</span>
+          <span fxFlex="60%">{{recurringDepositProduct.preClosurePenalInterest}} &nbsp; &nbsp; % &nbsp; &nbsp; On
+            &nbsp;&nbsp;
+            {{recurringDepositProduct.preClosurePenalInterestOnType.value}}</span>
+        </div>
+
+        <div fxFlexFill *ngIf="recurringDepositProduct.minBalanceForInterestCalculation">
+          <span fxFlex="40%">Balance Required For Interest Calculation:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.minBalanceForInterestCalculation | number }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Is withhold tax Applicable:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.withHoldTax }}</span>
+        </div>
+
+        <div fxFlexFill *ngIf="recurringDepositProduct.withHoldTax">
+          <span fxFlex="40%">Withhold Tax group:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.taxGroup.name }}</span>
+        </div>
+
+
+        <h3 class="mat-h3" fxFlexFill>Interest Rate Chart</h3>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <div fxFlexFill>
+          <span fxFlex="60%" style="float:right;">
+            <a
+              href="#/interestratecharts/{{recurringDepositProduct.id}}/{{recurringDepositProduct.name}}/recurringdepositproduct">View
+              / Edit Interest Rate Charts</a>
+          </span>
+        </div>
```

Comment:
This is not working properly. Instead of this please update the display as specified above.

Reply:
Earlier the page wasn't implemented as it was common for the fixed deposit product and recurring deposit product. Now I have created a new component and implemented so that can also be used for Fixed Deposit Product. Please have a look and let me know if any changes required.